airflow/traces/otel_tracer.py

```diff
@@ -57,7 +57,7 @@ class OtelTrace:
     When OTEL is enabled, the Trace class will be replaced by this class.
```

Comment:
This comment is here to block merging while it is open; feel free to resolve this after (and only after) @howardyoo gives his blessing.